containers/session-containers/skaha-desktop/Dockerfile.centos.xfce.vnc

```diff
@@ -115,16 +103,18 @@ RUN useradd -m -g guest -u 30001 guest && \
     ln -s /home/guest/.ssl /headless/.ssl
 
 # Add scripts that launch sofware containers
-ADD tmp/*.sh /usr/local/bin/
 ADD software-scripts/*.sh /usr/local/bin/
+RUN rm /usr/local/bin/dragons.sh
+RUN rm /usr/local/bin/nifty.sh
 
 # Add desktop icon files
 RUN mkdir -p /headless/.icons
 COPY software-scripts/icons/* /headless/.icons/
 
 # Add desktop links to those scripts
 COPY software-scripts/*.desktop /usr/share/applications/
-COPY tmp/*.desktop /usr/share/applications/
+RUN rm /usr/share/applications/DRAGONS.desktop
+RUN rm /usr/share/applications/nifty.desktop
```

Comment:
And remove this section?  (lines 114-117)

Reply:
Yes, can remove these lines after the two apps have been updated to work with generic .sh.

Reply:
There are .desktop files for other applications in software-scripts/., e.g. gedit.desktop, terminal.desktop. Unless we want to generate .desktop files for these apps, we need to keep line 115.

Reply:
Removed lines 116 and 117 and associated .desktop files.